staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -60,8 +60,8 @@ type ApplyOptions struct {
 	ForceConflicts  bool
 	FieldManager    string
 	Selector        string
-	DryRun          bool
 	ServerDryRun    bool
+	DryRunStrategy  cmdutil.DryRunStrategy
```

Comment:
Line 402 in this file, you're still using `ServerDryRun` as being the authoritative source of dry-run, which is wrong. I suspect you should remove this variable and merge it with the DryRunStrategy early, so that there is no source of confusion later on.

Reply:
It won't be easy unfortunately. We could make sync it the other way to make sure that if something uses it, it's not going to break.

Reply:
Can't wait for this flag to go for real.

Reply:
Oops fixed! Removed the struct value, which is merged early, so we don't use it anywhere anymore